### ISSUE TITLE
check_libyaml.c:2:18: fatal error: yaml.h: No such file or directory compilation terminated

### ISSUE THREAD

**Issue by blueandgold** [GoogleCloudPlatform/forseti-security#940]
When running ```python setup.py install``` on ubuntu, installation of pyyaml is throwing this error from check_libyaml.c file.

After looking into this issue, this is not critical as pyyaml will fall back to the native python parser.

But to fix this, it looks like either ```libpython-dev``` or ```python-dev``` needs to be installed.
https://github.com/aws/aws-cli/issues/2290
(Our documentation already mentions python-dev.)

Or perhaps this:
https://github.com/NVIDIA/DIGITS/issues/774
sudo apt-get install libyaml-cpp-dev

How to test:
http://rmcgibbo.github.io/blog/2013/05/23/faster-yaml-parsing-with-libyaml/

```
Searching for PyYAML==3.12
Reading https://pypi.python.org/simple/PyYAML/
Downloading https://pypi.python.org/packages/6b/f0/a0250248ea260d55748fff586d89a32afbb22656f4498b08d2636a48d4ec/PyYAML-3.12.zip#md5=aa5e762e79b19bb80dbcf04f82178832
Best match: PyYAML 3.12
Processing PyYAML-3.12.zip
Writing /tmp/easy_install-6j71iG/PyYAML-3.12/setup.cfg
Running PyYAML-3.12/setup.py -q bdist_egg --dist-dir /tmp/easy_install-6j71iG/PyYAML-3.12/egg-dist-tmp-qzMjPm
build/temp.linux-x86_64-2.7/check_libyaml.c:2:18: fatal error: yaml.h: No such file or directory
compilation terminated.

libyaml is not found or a compiler error: forcing --without-libyaml
(if libyaml is installed correctly, you may need to
 specify the option --include-dirs or uncomment and
 modify the parameter include_dirs in setup.cfg)
zip_safe flag not set; analyzing archive contents...
Moving PyYAML-3.12-py2.7-linux-x86_64.egg to /home/ep/.environments/forseti-security/lib/python2.7/site-packages
Adding PyYAML 3.12 to easy-install.pth file

Installed /home/ep/.environments/forseti-security/lib/python2.7/site-packages/PyYAML-3.12-py2.7-linux-x86_64.egg
```

**Comment by mirons-google** [comment]
I incorrectly had thought this was resolved, but it now connects to #1094

**Comment by mirons-google** [comment]
Resolved in #1094
